backend/alerts/integrations/microsoft-teams/messages_test.go

```diff
@@ -0,0 +1,225 @@
+package microsoft_teams
```

Comment:
I'm taking a leaf out of the example set by the discord implementation.
This is probably the fastest and most effective way to test the ms teams
bot.

Successfully running this test validates about 60+% of the work for this PR.